content/03-reference/01-tools-and-interfaces/03-prisma-migrate.mdx

```diff
@@ -71,13 +71,13 @@ prisma migrate save --experimental
 prisma migrate up --experimental
 ```
 
-The first command _saves_ a new migration to the file system and updates the `_Migration` table. It stores information about the migration in a dedicated directory inside the `migrations` directory. Each migration that is stored has its own `README.md` file which contains detailled information about the migration (e.g. the generated SQL statements which will be executed when you run `prisma migrate up`).
+The first command _saves_ a new migration to the file system and updates the `_Migration` table. It stores information about the migration in a dedicated directory inside the `prisma/migrations` directory of your project. Each migration that is stored has its own `README.md` file which contains detailed information about the migration (e.g. the generated SQL statements which will be executed when you run `prisma migrate up`).
```

Comment:
Let me know if you agree or disagree with the small change. The sentence was a little confusing because of multiple uses of the words migration/migrations and directory, so I just thought this makes it a little clearer for someone who is totally new to Prisma. Let me know if you think it's not necessary, though.